apps/src/lib/config/ethereum.rs

```diff
@@ -8,22 +8,23 @@ pub const DEFAULT_ORACLE_RPC_ENDPOINT: &str = "http://127.0.0.1:8545";
 
 #[derive(Clone, Debug, Serialize, Deserialize)]
 pub struct Config {
+    /// Whether a managed `geth` subprocess should be started or not.
+    pub geth_startup: bool,
     /// The Ethereum JSON-RPC endpoint that the Ethereum event oracle will use
     /// to listen for events from the Ethereum bridge smart contracts
     pub oracle_rpc_endpoint: String,
     /// If this is set to `true`, then instead of the oracle listening for
     /// events at a Ethereum JSON-RPC endpoint, an endpoint will be exposed by
     /// the ledger for submission of Borsh-serialized
     /// [`EthereumEvent`]s
-    #[cfg(not(feature = "eth-fullnode"))]
     pub oracle_event_endpoint: bool,
 }
```

Comment:
see my comment below in `apps/src/lib/node/ledger/mod.rs` regarding oracle startup